server-spi-private/src/main/java/org/keycloak/authentication/InitiatedActionSupport.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.keycloak.authentication;
+
+/**
+ *
+ * @author Stan Silvert
+ */
+public enum InitiatedActionSupport {
+    SUPPORTED, NOT_SUPPORTED, CONSENT_REQUIRED
```

Comment:
I would remove CONSENT_REQUIRED as no support is added for it.

Reply:
OK.  Hopefully, it will be added back soon.